lib/py/images/images.py

```diff
@@ -68,6 +68,20 @@ class MagicException(Exception):
     def __init__(self, magic):
         self.magic = magic
 
+# A helper function to decode base64 data
+# based on the version of the python interpreter
+def decode_base64_data(data):
```

Comment:
@behouba Would you be able to make this comment a [docstring](https://peps.python.org/pep-0257/)?
For example:
```python
def decode_base64_data(data):
    """A helper function to decode base64 data."""
```

Reply:
Done!